src/Me/components/Textarea/Textarea.js

```diff
@@ -12,7 +12,7 @@ const StyledTextarea = styled.textarea`
   padding: 7px 12px 6px 8px;
   color: #4f4f4f;
   ::placeholder {
-    color: #898889;
+    color: ${props => (props.invalid ? '#ff5f58' : '#898889')};
```

Comment:
Does it make sense to convert both of these over to the CSS variables in `index.css`?

Reply:
done